hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -1038,6 +1038,9 @@
   public static final String HBASE_REGION_SPLIT_POLICY_KEY =
     "hbase.regionserver.region.split.policy";
 
+  public static final String HBASE_REGION_SPLIT_USE_HFILELINK_ENABLED =
```

Comment:
Do we really need this flag? For me I prefer we just do it by default. You can send an email to the dev list to see if there are any objections, if not, let's just do it!

Reply:
Thanks, I'll send an email to ask for advice.